### PR TITLE
Added type.highway.busway translations

### LINKED ISSUE #3996: Busway is displaying type.highway.busway
Since #3739 (busways are shown on map), there is no title for each busway (displaying type.highway.busway per image).

If pointed out where I can edit this, I can glad fix this.

IMG_6745B221FFC6-1

### PR DESCRIPTION
**Pull request by matheusgomesms**
Closes #3996

I did not feel comfortable in using automatic translation, given this is a very specific term (and varies by countries with same language, such as Spanish). Therefore, I used mainly the terms displayed on Wikipedia.

PS.: could someone please regenerate the strings? :)

Signed-off-by: Matheus Gomes

### CODE REVIEW

**Review by vng — APPROVED** [review]

**Comment by biodranik** [comment]
There is an issue with this approach, and I don't know the right solution. Translations should be different per-country, in addition to per-language.

**Comment by matheusgomesms** [comment]
> There is an issue with this approach, and I don't know the right solution. Translations should be different per-country, in addition to per-language.

I know... According to my research, the terms I used were the most common in different countries. For now, we could leave this specific term to be decided by local translators?

File: data/strings/types_strings.txt [hunk 1]
```diff
@@ -9247,7 +9247,21 @@
     ja = 馬道(トンネル)
     pl = Tunel drogowy dla koni
     pt-BR = Túnel para cavaleiros
-
+    
+  [type.highway.busway]
```
Comment (biodranik) [inline data/strings/types_strings.txt:9251]: What about this?

```
en = Dedicated Bus Lane
ar = ﺕﻼﻓﺎﺤﻠﻟ ﺺﺼﺨﻣ ﺮﻤﻣ
be = Выдзеленая аўтобусная паласа
bg = Обособена бус лента
ca = Carril bus dedicat
cs = Vyhrazený autobusový pruh
da = Dedikeret busbane
de = Eigene Busspur
el = Αφιερωμένη λωρίδα λεωφορείων
es = Carril exclusivo para autobuses
et = Spetsiaalne bussirada
eu = Autobus-errei espezifikoa
fa = ﺱﻮﺑﻮﺗﺍ ﯽﺻﺎﺼﺘﺧﺍ ﻂﺧ
fi = Oma bussikaista
fr = Voie réservée aux autobus
he = ידועיי סובוטוא ביתנ
hu = Kijelölt buszsáv
id = Jalur khusus bus
it = Corsia dedicata agli autobus
ja = バス専用レーン
ko = 버스전용차로
mr = समर्पित बस लेन
nb = Dedikert bussfelt
nl = Eigen busbaan
pl = Wyznaczony pas dla autobusów
pt = Faixa exclusiva para ônibus
pt-BR = Faixa exclusiva para ônibus
ro = Banda de autobuz dedicata
ru = Выделенная автобусная полоса
sk = Vyhradený autobusový pruh
sv = Dedikerad bussfil
sw = Njia ya basi iliyotengwa
th = ช่องเดินรถเฉพาะ
tr = Özel otobüs şeridi
uk = Виділена автобусна смуга
vi = Làn đường dành riêng cho xe buýt
zh-Hans = 巴士专用道
zh-Hant = 巴士專用道
```

Comment (matheusgomesms) [reply data/strings/types_strings.txt:9251]: Hi Alexander,

Bus lanes are different than busways, thus [mapped in a different manner](https://wiki.openstreetmap.org/wiki/Bus_lanes). _Busway_ on OSM is a generic name for bus-only roads, commonly named as BRT or Metrobus, and mapped separately (`highway=busway`). On the other hand, bus lanes are mapped directly on roads, adding solely the lane information.

Therefore, I would not suggest this change on translation.

Comment (biodranik) [reply data/strings/types_strings.txt:9251]: What about this?

```
en = Dedicated bus road
ar = ﺺﺼﺨﻣ ﺕﻼﻓﺎﺣ ﻖﻳﺮﻃ
be = Выдзеленая аўтобусная дарога
bg = Обособен автобусен път
ca = Carretera dedicada a l'autobús
cs = Vyhrazená autobusová silnice
da = Dedikeret busvej
de = Eigene Busstraße
el = Αφιερωμένος δρόμος λεωφορείων
es = Carretera exclusiva para autobuses
et = Spetsiaalne bussitee
eu = Autobus errepidea
fa = ﺱﻮﺑﻮﺗﺍ ﯽﺻﺎﺼﺘﺧﺍ ﻩﺩﺎﺟ
fi = Oma bussitie
fr = Route réservée aux bus
he = ידועיי סובוטוא שיבכ
hu = Kijelölt buszút
id = Jalan khusus bus
it = Strada dedicata agli autobus
ja = バス専用道路
ko = 버스전용도로
mr = समर्पित बस रस्ता
nb = Dedikert bussvei
nl = Speciale busweg
pl = Dedykowana droga autobusowa
pt = Estrada de ônibus dedicada
pt-BR = Estrada de ônibus dedicada
ro = Drum dedicat autobuzului
ru = Выделенная автобусная дорога
sk = Vyhradená autobusová cesta
sv = Dedikerad bussväg
sw = Barabara ya basi iliyotengwa
th = ถนนทางเดินรถโดยเฉพาะ
tr = Özel otobüs yolu
uk = Виділена автобусна дорога
vi = Đường dành riêng cho xe buýt
zh-Hans = 巴士专用道
zh-Hant = 巴士專用道
```

Comment (matheusgomesms) [reply data/strings/types_strings.txt:9251]: Much better! I don't have any problem with this!

For Portuguese, I would use on this case:

pt = Via exclusiva de autocarro
pt-BR = Via exclusiva de ônibus

Shall I updated everything or not?

Comment (biodranik) [reply data/strings/types_strings.txt:9251]: These were quick and dirty Google translations. We're overloaded with issues, so your help is really appreciated 🙏. You may polish them with DeepL and use some regional names too if it feels appropriate.

Comment (matheusgomesms) [reply data/strings/types_strings.txt:9251]: Great, I updated them! Just need to regenerate, then all translators can review on their language! Thanks

**Pull request merged**